README.md

```diff
@@ -1,2 +1,89 @@
 # nomad-plugin-template
 A template repository for creating a repository with a NOMAD plugin package.
+
+> [!WARNING]
+> The template is not yet ready for production usage.
+
+
+## Getting started
+
+1. Click on the `Use this template` button and create a new plugin repository. The form will ask you to fill out the name for the new plugin repository.
+
+2. In the newly created repository, start a new Github Codespace and generate the plugin structure.
+
+Run the following command to create a new Nomad plugin project using cookiecutter-nomad-plugin:
+
+```sh
+cruft create https://github.com/FAIRmat-NFDI/cookiecutter-nomad-plugin
```

Comment:
This doesn't work yet, the default branch needs to be changed in the repository settings there.